src/main/java/ch/njol/skript/expressions/ExprPlayerProtocolVersion.java

```diff
@@ -0,0 +1,49 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import org.bukkit.entity.Player;
+import org.eclipse.jdt.annotation.Nullable;
+
+public class ExprPlayerProtocolVersion extends SimplePropertyExpression<Player, Integer> {
+
+	static {
+		if (Skript.classExists("com.destroystokyo.paper.network.NetworkClient")) {
+			register(ExprPlayerProtocolVersion.class, Integer.class, "[the] [client] protocol version", "players");
+		}
+	}
+
+	@Override
+	public @Nullable Integer convert(Player player) {
```

Comment:
The `@Nullable` tag goes under `@Override` and above the public method. Everything else looks good to me. We don't have to worry about default expressions here as we're removing an old syntax that used the existing syntax. So it's good as it other than this micro code standard change.